index.html

```diff
@@ -4,484 +4,528 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Job search</title>
-    <link rel="stylesheet" href="./src/css/normalizes.css" />
-    <link rel="stylesheet" href="./src/css/resets.css" />
+    <title>Jobply - job search</title>
+    <link type="Image/x-icon" href="./src/favicons/favicon.ico" rel="icon" />
```

Comment:
```suggestion
    <link type="Image/x-icon" href="favicon.ico" rel="icon" />
```
1. Replace the favicon to the root of your project
2. Read [this guide](https://vc.ru/u/1185570-mihail-gok/586763-kak-sozdavat-ikonki-saytov-v-2023-godu-vse-o-favicon) and follow it to add more files that may be needed

Reply:
i did it, but it doesnt work